bip-0339.mediawiki

```diff
@@ -41,9 +41,10 @@ announcing and fetching transactions.
 
 # A new wtxidrelay message is added, which is defined as an empty message where pchCommand == "wtxidrelay".
 # The protocol version of nodes implementing this BIP must be set to 70016 or higher.
-# The wtxidrelay message must be sent in response to a VERSION message from a peer whose protocol version is >= 70016, and prior to sending a VERACK.
-# A new inv type MSG_WTX (0x00000005) is added, for use in both INV messages and GETDATA requests, indicating that the hash being referenced is a transaction's wtxid. In the case of GETDATA requests, MSG_WTX implies that the transaction being requested should be serialized with witness as well, as described in BIP 144.
-# After a node has sent and received a "wtxidrelay" message to/from a given peer, the node is required to use the MSG_WTX inv-type when announcing transactions to that peer, or requesting announced transactions from that peer.
+# The wtxidrelay message MUST be sent in response to a version message from a peer whose protocol version is >= 70016, and prior to sending a verack. A wtxidrelay message received after a verack message MUST be ignored or treated as invalid.
+# A new inv type MSG_WTX (0x00000005) is added, for use in both inv messages and getdata requests, indicating that the hash being referenced is a transaction's wtxid. In the case of getdata requests, MSG_WTX implies that the transaction being requested should be serialized with witness as well, as described in BIP 144.
+# After a node has received a wtxidrelay message from a peer, the node MUST use the MSG_WTX inv-type when announcing transactions to that peer.
```

Comment:
s/inv-type/inv type/ ("inv type" is also used in line 45 immediately above)

Reply:
Good idea. Standardized to 'inv type'